pkgs/servers/web-apps/slskd/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, stdenv
+, fetchFromGitHub
+, fetchurl
+, unzip
+, dotnetCorePackages
+, buildDotnetModule
+, mono
+}:
+let
+  pname = "slskd";
+  version = "0.17.5";
+
+  # TODO build this via npm
+  wwwroot = stdenv.mkDerivation {
+    inherit version;
+    pname = "${pname}-wwwroot";
+
+    src = fetchurl {
```

Comment:
Maybe it's better to use the same src being used in buildDotnetModule?

Reply:
Yeah it would definitely be better. But then, we have to build the web ui ourselves: I didn't try yet, and you were stuck on this in your gist, right?

Reply:
See my comment below.